service/core/kafka/server/src/main/scala/com/lightbend/lagom/internal/broker/kafka/Producer.scala

```diff
@@ -50,22 +53,14 @@ private[lagom] object Producer {
       offsetStore
     )
 
-    val backoffPublisherProps = BackoffOpts
-      .onStop(
-        publisherProps,
-        s"producer",
-        producerConfig.minBackoff,
-        producerConfig.maxBackoff,
-        producerConfig.randomBackoffFactor
-      )
-      .withDefaultStoppingStrategy
+    val entityIds = tags.toSet
 
```

Comment:
The main change here was removing this, now obsolete, BackOff supervisor. When the stream in `TopicProducer` crashes the `BackoffSource` logic will take care of restarting the Source without rebuilding the whole actor. Because of this change, the `Source` will immediately start instead of having to wait for a new `EnsureActive` ping. 🎉